nestkernel/conn_builder.cpp

```diff
@@ -369,13 +371,29 @@ nest::ConnBuilder::change_connected_synaptic_elements( index sgid,
 void
 nest::ConnBuilder::connect()
 {
+  if ( symmetric_ && not supports_symmetric() )
+    throw NotImplemented(
+      "So far Connect doesn't support symmetric connections for this "
```

Comment:
I would even rephrase as "This connection rule does not support symmetric connections" because the function might be called from other sources than just plain `Connect`. It might also help to include the name of the rule, if that's accessible here. @hannahbos, can you help here?